meancoach_project/apps/metrics/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import Metric, MetricRecord
+
+admin.site.register(Metric)
+admin.site.register(MetricRecord)
```

Comment:
this adds the models to the "django admin" which is a generic place in Django where you can see your database models and have auto-generated forms to edit them.